baseapp/baseapp.go

```diff
@@ -161,6 +162,9 @@ type BaseApp struct {
 	//
 	// SAFETY: it's safe to do if validators validate the total gas wanted in the `ProcessProposal`, which is the case in the default handler.
 	disableBlockGasMeter bool
+
+	// Optional alternative tx runner, used for block-stm parallel transaction execution. If nil, default txRunner is used.
```

Comment:
this comment seems untrue, since in `executeTxsWithExecutor`, if its nil we set it to the bstm one

Reply:
oh. that seems confusing. can we have the non-blockstm runner live outside the blockstm package?